config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -57,6 +57,20 @@ presets:
       items:
       - key: bootstrapper-key
         path: bootstrapper-key
+- labels:
+    preset-cluster-api-provider-vsphere-gcs-creds: "true"
+  volumes:
+  - name: cluster-api-provider-vsphere-gcs-creds
+    secret:
+      secretName: cluster-api-provider-vsphere-gcs-prow
+      items:
+      - key: keyfile.json
+        path: keyfile.json
+        mode: 288
```

Comment:
For anyone else reviewing, the UNIX permission `288` is the decimal representation of the octal value `0440`. In [other words...](http://permissions-calculator.org/decode/0440/)